doc/htmldoc/synapses/connection_management.rst

```diff
@@ -256,6 +256,107 @@ must be ``True``.
+:py:func:`.TripartiteConnect` function:
+
+.. code-block:: python
+
+    TripartiteConnect(pre, post, third, conn_spec)
+    TripartiteConnect(pre, post, third, conn_spec, syn_specs)
+
+``pre``, ``post``, and ``third`` are ``NodeCollections``, defining the nodes of
+origin (`sources`) and termination (`targets`) as well as the third
+factor to be included. Details of the connections created depend on
+the connection rule used.
+
+``conn_spec`` must be provided and must specify a tripartite
+connection rule.
+
+``syn_specs`` is a dictionary of the form
+
+.. code-block:: python
+
+    {"primary": <syn_spec>,
+     "third_in": <syn_spec>,
+     "third_out": <syn_spec>}
+
+where the individual ``syn_spec`` elements follow the same rules as
+for the :py:func:`.Connect` function. Any of the three elements can be
+left out, in which case the synapse specification defaults to
+``"static_synapse"``. The ``"primary"`` synapse specification applies
+to connections from ``pre`` to ``post`` nodes, the ``"third_in"``
+specification to connections from ``pre`` to ``third`` nodes and
+the ``"third_out"`` specification to connections from ``third`` to
+``post`` nodes.
+
+tripartite_bernoulli_with_pool
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For each possible pair of nodes from ``A`` and ``B``, a connection is
+created with probability ``p_primary``, and these connections are
+called "primary" connections. For each primary connection, a
+third-party connection pair involving a node from ``C`` (a third
+``NodeCollection``) is created with the conditional probability
+``p_third_if_primary``. This connection pair includes a connection
+from the node from ``A`` (i.e., the source) to the node from ``C`` in
+question, and a connection from this node from ``C`` to the node from
+``B`` (i.e., the target). The node from ``C`` to connect to is chosen
+at random from a pool, a subset of the nodes in ``C``. By default,
+this pool is all of ``C``.
+
+Pool formation is controlled by parameters ``pool_type``, which can be ``"random"``
+(default) or ``"block"``, and ``pool_size`` which must be between 1
+and the size of ``C`` (default). For random pools, for each node from
+``B``, ``pool_size`` nodes from ``C`` are chosen randomly without
+replacement.
+
+For block pools, two variants exist. Let ``n(B)`` and ``n(C)`` be the number of
+nodes in ``B`` and ``C``, respectively. If ``pool_size == 1``, the
+first ``n(B)/n(C)`` nodes in ``B`` are assigned the first node in
+``C`` as their pool, the second ``n(B)/n(C)`` nodes in ``B`` the
+second node in ``C`` and so forth. In this case, ``n(B)`` must be a
+multiple of ``n(C)``. If ``pool_size > 1``, the first ``pool_size``
+elements of ``C`` are the pool for the first node in ``B``, the
+second ``pool_size`` elements of ``C`` are the pool for the second
+node in ``B`` and so forth. In this case, ``n(B) * pool_size == n(C)``
+is required.
+
+.. code-block:: python
+
+    A = nest.Create('aeif_cond_alpha_astro', 10)
```

Comment:
Change this example so that it corresponds to one of the figure panels, at best the left one.